src/middlewared/middlewared/etc_files/mail/aliases.mako

```diff
@@ -5,7 +5,7 @@
 
     users = filter_list(render_ctx['user.query'], [['email', 'nin', ['', None]]])
 
-    with open(os.path.join('/conf/base/etc', '' if IS_LINUX else 'mail', 'aliases'), 'r') as f:
+    with open(os.path.join('/conf/base/etc/aliases'), 'r') as f:
```

Comment:
Another nit, but you can remove the `os.path.join` since it's no longer joining anything 😄